app/org/sagebionetworks/bridge/models/healthdata/HealthDataAttachment.java

```diff
@@ -0,0 +1,30 @@
+package org.sagebionetworks.bridge.models.healthdata;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import org.sagebionetworks.bridge.dynamodb.DynamoHealthDataAttachment;
+import org.sagebionetworks.bridge.json.BridgeTypeName;
+
+/**
+ * Metadata record about health data attachments. This is used to keep track of attachments in external storage
+ * (generally S3) and the actual health data record.
+ */
+@BridgeTypeName("HealthDataAttachment")
+@JsonDeserialize(as = DynamoHealthDataAttachment.class)
+public interface HealthDataAttachment {
+    /**
+     * Attachment ID. This is used as both the unique attachment ID and the S3 key in the attachments bucket to
+     * guarantee filename uniqueness in S3.
+     */
+    String getId();
+
+    /** Record ID of the health data record this file is attached to. */
+    String getRecordId();
```

Comment:
The Cardio 6-Minute Walk Test has multiple JSON blobs that we'd want to save as separate attachments. It's not really meaningful to try to combine them into a single attachment. Plus, in the long term, if clients are uploading attachments directly instead of in upload bundles, it makes more sense to store them as separate attachments.